lms/djangoapps/bulk_email/migrations/0003_config_model_feature_flag.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import migrations, models
 import django.db.models.deletion
 from django.conf import settings
+from django.db import migrations, models
```

Comment:
Thanks @robrap and @doctoryes. I've added migrations and envs to the skip configuration.